webpages/settings/components/addon-setting.js

```diff
@@ -149,6 +150,10 @@ export default async function ({ template }) {
         this.addonSettings[this.setting.id] = newValue;
         this.updateSettings();
       },
+      checkOverflowing(e) {
+        this.isOverflowing = window.innerWidth - e.clientX < 250;
```

Comment:
A few issues:
- The mouse could be at a slightly different location on different function calls, causing the tooltip to pop out inconsistently.
  - Could we use `getBoundingClientRect()` instead of `MouseEvent.clientX`?
- There is still a scroll bar along the bottom of the page before this function is called to change the popout direction.
  - This could be fixed by setting `display: none` instead of `visibility: hidden` on the tooltip.